docs/variants/msi_z690/recovery.md

```diff
@@ -7,6 +7,9 @@ hardware configurations, the Dasharo firmware may not boot correctly (i.e.
 we will have "bricked" the platform). In such a case, the recovery procedure can
 reinstall the original firmware from the board manufacturer.
 
+There are two documented recovery methods: using a CH341A programming kit or an
+RTE external programmer.
```

Comment:
```suggestion
RTE.
```

Both CH341A and RTE are external SPI programmers

Maybe linking to those sections would be useful if this document will grow in future

Reply:
I think that's a good idea, I'll include it in my next PR.